src/Signature/SignatureV4.php

```diff
@@ -82,7 +82,9 @@ public function presign(
         $expires,
         array $options = []
     ) {
-        $startTimestamp = isset($options['start_time']) ? $this->convertToTimestamp($options['start_time']) : time();
+        $relativeTimeBase = isset($options['relativeTimeBase']) ? $options['relativeTimeBase'] : time();
+        $expiresTimestamp = $this->convertToTimestamp($expires,$relativeTimeBase);
```

Comment:
Thanks for making these updates @daum!

It seems like I was unclear previously when talking about this section, apologies. I don't think there needs to be a separate `relativeTimeBase` *option*, but instead use the `$startTimestamp` we already have for this line passed to `$relativeTimeBase` of `convertToTimestamp` when generating `$expiresTimestamp`. I can't think of any cases where someone would want `'relativeTimeBase'` to not be the same as `'start_time'`, let me know if I'm missing something.

And a small formatting update: please add a space after the `,` in this function call.